crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -41,6 +41,19 @@ def roles_for_select(selected)
     )
   end
 
+  # Possible values for nodes default file system
+  def fs_for_select(selected)
+    options_for_select(
+      [
+        ["ext3", "ext3"],
```

Comment:
Do we really want ext3?

Should we put some words of caution about the non-ext4 fs (or at least, btrfs -- since it's totally untested for controllers or kvm compute nodes)